.github/workflows/e2e-full.yml

```diff
@@ -0,0 +1,51 @@
+name: e2e-full
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 */3 * *"  # Run every 3 days at midnight UTC
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  FOUNDRY_PROFILE: ci
+  ETH_TESTNET_TYPE: pos
+  SP1_PROVER: network
+  ETHEREUM_POS_NETWORK_PRESET: mainnet
+permissions:
+  contents: read
+
+jobs:
+  generate-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.get-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Generate Test Matrix
+        id: get-matrix
+        uses: ./.github/actions/e2e-test-matrix
+        with:
+          skip-tests: 'TestWithRelayerTestSuite/Test_2_ConcurrentRecvPacketToEth_Groth16,TestWithRelayerTestSuite/TestMultiPeriodClientUpdateToCosmos'
+
+  e2e:
+    needs: generate-matrix
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.generate-matrix.outputs.matrix) }}
+    name: ${{ matrix.test }}
+    runs-on: depot-ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Set up E2E environment"
```

Comment:
Some name use quotations and some names don't. I think we should just not use quotations but I'm ok with either for names as long as we are consistent. Ditto everywhere else.